website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and to ensure up-to-date data products, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models for you whenever a pull request (PR) merges, making sure the latest changes are in production (such as modifications to code that generates the tables). You don't need to wait for the next scheduled job to run. 
+
+You can also implement continuous integration (CI) in dbt Cloud, which can help further to reduce the time it takes to push changes to production and improve code quality. To learn more, refer to [Continuous integration in dbt Cloud](/docs/deploy/continuous-integration). 
+
+## How merge jobs work
+
+When you set up merge jobs, dbt Cloud listens for notifications from your Git provider indicating that a PR has been merged. When dbt Cloud receives one of these notifications, it enqueues a new run of the merge job.
```

Comment:
Do we need to call out which Git providers this works for? We can only listen to ADO, Github, and Gitlab because they are the only ones we have integrations for. You can do this with others, but you need to have "action" on the Git provider side to trigger the job via the API.

Reply:
we don't need to here since this is an "about/overview" page. the merge jobs "how to" page has a prerequisites section that already calls this out

i'll add a link here tho for customers who want to dive deeper 👍